### ISSUE TITLE
FileNotFoundError on Test the Trained Model

### ISSUE THREAD

**Issue by DashBarkHuss** [TheLastBen/fast-stable-diffusion#568]
When I run `Test The Trained Model` I get:
```Already up to date.
---------------------------------------------------------------------------
FileNotFoundError                         Traceback (most recent call last)
[<ipython-input-24-20272899b2d6>](https://localhost:8080/#) in <module>
    160   srv= getoutput('cat /content/srvr.txt')
    161 
--> 162   for line in fileinput.input('/usr/local/lib/python3.7/dist-packages/gradio/blocks.py', inplace=True):
    163     if line.strip().startswith('self.server_name ='):
    164         line = f'            self.server_name = "{srv[8:]}"\n'

1 frames
[/usr/lib/python3.7/fileinput.py](https://localhost:8080/#) in _readline(self)
    337                     pass
    338                 # The next few lines may raise OSError
--> 339                 os.rename(self._filename, self._backupfilename)
    340                 self._file = open(self._backupfilename, self._mode)
    341                 try:

FileNotFoundError: [Errno 2] No such file or directory: '/usr/local/lib/python3.7/dist-packages/gradio/blocks.py' -> '/usr/local/lib/python3.7/dist-packages/gradio/blocks.py.bak'
```

What could cause this?

**Comment by TheLastBen** [comment]
did you run the dependencies cell ?

**Comment by DashBarkHuss** [comment]
Yes, I [did](https://ibb.co/0rVCFn20. 

I didn't run the **Training** cell or the  **Downloading the model** cell. Are those important if you already trained the model?

**Comment by DashBarkHuss** [comment]
I started the whole colab over and it worked. I just refreshed but for some reason, this time I refreshed I noticed something weird: This time when Ii started over it had disconnected from my Gdrive. It asked to connect again. When it connected this time it was connected to my actual GDrive. I could see other files in my Gdrive, not from Colabs. Whereas before, I could not see these other files. So maybe it had been connecting to some other GDrive? or there was some conflict?
